xmodule/templates/html/zooming_image.yaml

```diff
@@ -0,0 +1,230 @@
+        </a>
+      </div>
+      <!-- MODIFY HERE: end -->
+      <!-- DO NOT REMOVE: start -->
+      <script type="text/javascript">//<![CDATA[
+        /*
+        jQuery loupeAndLightbox Plugin
+        * Version 1.1
+        * 12-11-2024
+        * Authors: M.Biscan, Adolfo R. Brandes
+        * requires jQuery 1.4.2
+        Copyright (c) 2010 M.Biscan
+
+        Permission is hereby granted, free of charge, to any person obtaining
+        a copy of this software and associated documentation files (the
+        "Software"), to deal in the Software without restriction, including
+        without limitation the rights to use, copy, modify, merge, publish,
+        distribute, sublicense, and/or sell copies of the Software, and to
+        permit persons to whom the Software is furnished to do so, subject to
+        the following conditions:
+
+        The above copyright notice and this permission notice shall be
+        included in all copies or substantial portions of the Software.
+
+        THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+        EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+        MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+        NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+        LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+        OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+        WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+        */
+        (function($) {
+          $.fn.loupe = function(options) {
```

Comment:
I'm guessing this also happened with the previous component, I'm not sure. But if I have two zooming image components, this would happen when zooming into one:

![image](https://github.com/user-attachments/assets/b9e66b68-c537-40e9-82ab-c48124a5fdf4)

I'm guessing this happens because the loupe is global to the window? I'm okay with having this caveat.

Reply:
Nice catch!  Yeah, this would happen with the previous one, too, but I'll see if I can do anything about it - the whole point is that we can now fix bugs on it. ;)

Reply:
Ok, fixed!  Took some trickery, but there can now be two or more instances on the same page.

Reply:
Thank you :)